proto/environment/command.proto

```diff
@@ -63,3 +63,31 @@ message EnableProjectCommand {}
 message DisableProjectCommand {}
 
 message ConvertTrialProjectCommand {}
+
+message CreateOrganizationCommand {
+  string name = 1;
+  string url_code = 2;
+  string description = 3;  // optional
+}
+
+message CreateTrialOrganizationCommand {
```

Comment:
What do you think about adding a flag in the `CreateOrganizationCommand` to create a trial project to reduce the APIs?

Reply:
Nice idea 😄.